iac-extensions/docker/src/test/resources/checks/SecretsGenerationCheck/drush.dockerfile

```diff
@@ -0,0 +1,126 @@
+FROM ubuntu:22.04 as build
+
+# no issue in non final stage
+RUN drush user:password username password
```

Comment:
Are we sure it is safe in non-final stage?
I'm thinking about two scenarios:
- a dockerfile which is called with the `--target` option, in that case any image can be the final stage
- a final stage which "extend" from it, with `FROM build`

Reply:
* About `--target` we should discuss it in the team
* About "extend" - it's ok, so long the generated file with the password is not copied to the final stage. But to detect that we need taint analysis. 🤷‍♂️